sympy/series/tests/test_series.py

```diff
@@ -17,6 +17,30 @@ def test_cos():
     assert e1 == e2
 
 
+def test_tan():
```

Comment:
Would it be better to merge all these tests and `test_sin` and `test_cos` into one `test_trigonometric`? Same for the multiple `test_exp` functions.

Reply:
Thanks, @0sidharth  , now its looks good..